### DISCUSSION TITLE
Release training/fine-tuning code?

### DISCUSSION THREAD

**Discussion by jaggzh** [openai/whisper#1758] [Ideas]
Can you guys kindly bestow the gift upon us of fine-tuning and training code (for us to use on our own systems)?
[Another edit] I *just* realized why you likely didn't.  Okay, perhaps I can contact someone and request it just for personal use for someone who breaths using a ventilator and no existing models have ever worked for her?  I'm in Los Angeles and can agree to an NDA/

**Comment by phineas-pta** [comment]
unfortunately, regarding model architecture + data availability + software integration, i dont think current open source speech recognition is suitable for patients in critical situations

u may want to look for other solutions

**Reply by jaggzh** [reply]
Who are you? Do you speak on behalf of openai/whisper?

**Reply by phineas-pta** [reply]
im just random stranger on internet, but dont expect openai to answer your question about whisper
